src/login/pages/LoginTemp.svelte

```
@@ -0,0 +1,228 @@
+                name="username"
+                value={login.username ?? ''}
+                type="text"
+                autofocus
+                autocomplete="username"
+                aria-invalid={messagesPerField.existsError('username')}
+              />
+              {#if messagesPerField.existsError('username')}
+                <div
+                  id="input-error"
+                  class={kcClsx('kcInputErrorMessageClass')}
+                  aria-live="polite">{@html kcSanitize(messagesPerField.getFirstError('username'))}</div
```

ESLint

[error] Replace `>{`@html`·kcSanitize(messagesPerField.getFirstError('username'))}</div` with `⏎················>⏎··················{`@html`·kcSanitize(messagesPerField.getFirstError('username'))}`

(prettier/prettier)